src/select.coffee

```diff
@@ -140,16 +153,39 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       @get('originalItemViewClass')
   .property 'showTooltip'
 
+  ###*
+  * Function to be called when we want to open the dropdown, i.e. we have to
+  * clean up the query, render the dropdown...
```

Comment:
I don't think our docs should have ellipses in them, especially for open source projects. The other items in that list may be obvious to you, but not others.